README.md

```diff
@@ -79,9 +79,9 @@ redocly lint openapi.yaml
 
 **Configure the rules** as you wish. Other API Linters use complicated identifiers like JSONPath, but Redocly makes life easy with simple expressions that understand the OpenAPI structure. You can either use the [built-in rules](https://redocly.com/docs/cli/rules) to mix-and-match your ideal API guidelines, or break out the tools to build your own.
 
-**Format the output** in whatever way you need, the `stylish` output is as good as it sounds, but if you need JSON or Checkstyle outputs to integrate with other tools, the lint command can output those too.
+**Format the output** in whatever way you need. The `stylish` output is as good as it sounds, but if you need JSON or Checkstyle outputs to integrate with other tools, the `lint` command can output those too.
 
-**Multiple files supported** so you don't need to bundle your API description to lint it, just point Redocly CLI at the "entry point" (e.g.: `openapi.yaml`) and it handles the rest.
+**Multiple files supported** so you don't need to bundle your API description to lint it. Just point Redocly CLI at the "entry point" (e.g.: `openapi.yaml`) and it handles the rest.
```

Comment:
This really is all one sentence, so either revert the change or use a semicolon instead.

Reply:
Semicolon it is!